admin-dev/themes/default/template/controllers/dashboard/helpers/view/view.tpl

```diff
@@ -119,7 +119,7 @@
 						<dd>{l s="Enhance your store with templates & modules" d='Admin.Dashboard.Feature'}</dd>
 					</dl>
 					<dl>
-						<dt><a href="http://forge.prestashop.com?utm_source=back-office&amp;utm_medium=dashboard&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content={if $host_mode}cloud{else}download{/if}" class="_blank">{l s="The Forge" d='Admin.Dashboard.Feature'}</a></dt>
+						<dt><a href="https://github.com/PrestaShop/PrestaShop/issues" class="_blank">{l s="The Forge" d='Admin.Dashboard.Feature'}</a></dt>
```

Comment:
do we keep the use of "Forge" name?

Reply:
IDK, what do you think @colinegin?

Reply:
Seems great to me !

Reply:
We need to change the wording then.

@LouiseBonnard do you think we should add it to "Admin.Dashboard.Feature" or should we reuse the existing one in "Admin.Navigation.Footer"?

Reply:
@eternoendless, where in the back office will this string be located: in the footer or on the dashboard?

Reply:
Both :)

Reply:
Well, I would localize it in _Admin.Global_ since it is not attached to a specific page but is likely to be found in the content of numerous pages.

Reply:
I'm on it, thanks